rs/sns/testing/src/utils.rs

```diff
@@ -0,0 +1,245 @@
+use anyhow::{anyhow, Context, Result};
+use dfx_core::config::model::network_descriptor::NetworkDescriptor;
+use dfx_core::identity::identity_manager::InitializeIdentity;
+use dfx_core::identity::IdentityManager;
+use futures::future::join_all;
+use ic_agent::Identity;
+use ic_agent::{
+    agent::route_provider::RoundRobinRouteProvider, identity::Secp256k1Identity, Agent,
+};
+use ic_base_types::CanisterId;
+use ic_base_types::PrincipalId;
+use ic_nervous_system_agent::nns::governance::list_neurons;
+use ic_nervous_system_agent::nns::sns_wasm::{get_latest_sns_version_pretty, get_sns_subnet_ids};
+use ic_nervous_system_agent::CallCanisters;
+use ic_nervous_system_common_test_keys::TEST_NEURON_1_ID;
+use ic_nns_common::pb::v1::NeuronId;
+use ic_nns_constants::{
+    canister_id_to_nns_canister_name, CYCLES_LEDGER_CANISTER_ID, CYCLES_MINTING_CANISTER_ID,
+    GOVERNANCE_CANISTER_ID, LEDGER_CANISTER_ID, LEDGER_INDEX_CANISTER_ID, LIFELINE_CANISTER_ID,
+    REGISTRY_CANISTER_ID, ROOT_CANISTER_ID, SNS_WASM_CANISTER_ID,
+};
+use ic_nns_governance_api::pb::v1::{ListNeurons, Neuron};
+use k256::SecretKey;
+use lazy_static::lazy_static;
+use reqwest::Client;
+use thiserror::Error;
+
+pub const ALL_NNS_CANISTER_IDS: [&CanisterId; 9] = [
```

Comment:
I would name this something else, since there's a constant in nns/constants called `ALL_NNS_CANISTER_IDS` and that's not actually what this is.  It's more about the canisters that are created for SNS testing.